### PR TITLE
Limericks Week in MIQ

### PR DESCRIPTION
**Pull request by d-m-u**
Blog post for Aug 1.

### CODE REVIEW

**Comment by JPrause** [comment]
@imtayadeway @jntullo @syncrou can you each have a look-see.
Thanks @d-m-u

File: site/_posts/2017-08-01-limericks-week-in-miq.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+title: "Limericks Week in ManageIQ"
+author: d-m-u, AllenBW
+date: 2017-08-01
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/O05SBdpl7Gc" frameborder="0" allowfullscreen></iframe>
```
Comment (syncrou) [inline site/_posts/2017-08-01-limericks-week-in-miq.md]: @d-m-u - It seems this embed url is not rendering correctly - at least when I'm viewing the html from the browser.

Comment (d-m-u) [inline site/_posts/2017-08-01-limericks-week-in-miq.md]: That's how all the things online say to do it though.

**Comment by jntullo** [comment]
LGTM 💯  😆

File: site/_posts/2017-08-01-limericks-week-in-miq.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+Refactors we adore  
+We’re just thrilled when updates are succinct.  
+
+Generic Objects page has been refreshed  
+Feature names are hence newly addressed  
+Fresh sections galore  
+Old views are done for   
+You’ll find show_list is now [at its best](https://github.com/ManageIQ/manageiq/pull/15611)!  
+
+A coder is who is from Hong Kong
```
Comment (Fryguy) [inline site/_posts/2017-08-01-limericks-week-in-miq.md]: Typo? I think there's an extra `is`.

Comment (d-m-u) [inline site/_posts/2017-08-01-limericks-week-in-miq.md]: Ah, thanks.

**Comment by Fryguy** [comment]
Amazing 😲

**Comment by JPrause** [comment]
Seems the reviews and edits have completed. 👍 
@Fryguy can you merge since Carol is on PTO

**Pull request merged**